### DISCUSSION TITLE
Hosting flowise as Azure web app from Docker, to embed in chatbot.

### DISCUSSION THREAD

**Discussion by mikkel-convai** [FlowiseAI/Flowise#2263] [Q&A]
Hi everybody
I hope this is the right forum to ask. 

I am trying to host flowise on Azure, so that i can use it for embedding later. I need to host it on Azure and be able to use it via an https connection. What is the best approach for this? 

I have tried to follow the documentation: https://docs.flowiseai.com/configuration/deployment/azure, to make a container instance. This worked, but I didn't get a https site, so i cant use that.
Then I've spend a bunch of time, trying to make it a web app from a container instance. I've followed this tutorial from vs code: https://code.visualstudio.com/docs/containers/app-service. 
I can make flowise run locally using docker and not changing anything, but whenever I try to make it a web app, i get it all setup and it looks fine, but when i go to the url, it just loads for ages and finally shows 504.0 GatewayTimeout. Ive checked the console in the browser, but that doesn't give  me anything.
When looking at the Azure Log Stream i get this:
![image](https://github.com/FlowiseAI/Flowise/assets/142321534/e37c02f3-a787-47f8-8200-b84001eaa8c4)

I hope someone can help. 
I am aware it would be good to add some code context or something like that, but I'm not quite sure what it should be, so let me know what i can add / where i can look to find the problem

**Comment by Nek-11** [comment]
Did you manage to solve this ?
